projects/demo/src/app/demo/pages/form-control-page/id-example/form-control-id-example.component.html

```diff
@@ -0,0 +1,22 @@
+<p>The provided id attribute is used whenever possible (avoiding duplicate identifier). When no id is provided, a generated
+    id is built using a default 'field' specifier.</p>
+
+<div paFormControl></div>
+<code>{{ noIdProvided }}</code>
+<div>formControl id: <b>{{autoId}}</b></div>
```

Comment:
you should use `strong` instead of `b` tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b